authority/authority.go

```diff
@@ -253,13 +253,44 @@ func (a *Authority) init() error {
 		}
 	}
 
+	// Initialize linkedca client if necessary. On a linked RA, the issuer
+	// configuration might come from majordomo.
```

Comment:
kinda weird to mention majordomo (not open source) in the documentation, no?

Reply:
Well the linkedca client is called Majordomo.